### DISCUSSION TITLE
Connexion au navigateur

### DISCUSSION THREAD

**Discussion by Arystho** [brackets-cont/brackets#321] [Feature Request]
Bonjour je n'arrive pas a tester mes commande avec google chrome ça me demande de relancer chrome et activer le débogage à distance comment on fais ça ??

**Comment by charlypa** [comment]
We recommend you download Phoneix Code which is the next generation of brackets
https://phcode.io/

For brackets as a workaround go to `file->Enable Experimental Live Preview`

**Reply by abose** [reply]
Chrome changed its security policy last year, and that prevents Brackets from activating its default live preview. Phoenix code is based on a new architecture and plays well with the new security policies.
